torch/distributed/tensor/parallel/style.py

```diff
@@ -51,8 +51,10 @@ class PairwiseParallel(ParallelStyle):
     We assume both input and output need to be replicate DTensors.
 
     .. warning::
-        PairwiseParallel only supports ``nn.Multihead Attention``,
-        ``nn.Transformer`` or even-number-layer MLP for now.
+        PairwiseParallel does not support ``nn.MultiheadAttention``,
+        ``nn.Transformer`` well at this moment. One workaround is to
+        use ``ColwiseParallel`` and ``RowwiseParallel`` directly. We recommend to
```

Comment:
nit: One workaround is to apply ``ColwiseParallel`` and ``RowwiseParallel`` to the components of transformer directly.